src/app/messages/en/messages.json

```diff
@@ -63,8 +63,10 @@
   "i18n:highlighter:edit-note:label": "Edit highlighted note",
   "i18n:highlighter:highlight:end": "End of {style} highlight",
   "i18n:highlighter:highlight:start": "Start {style} highlight",
+  "i18n:highlighter:highlight:start-selected": "Start selected {style} highlight",
```

Comment:
This doesn't seem to match the other messages?

Reply:
@jivey I'm not clear on what other messages should be matched and how. This is a message to indicate the start of the selected highlight (as opposed to other, not-currently-selected (orange) ones).

Reply:
Oh I was just looking at the other `i18n:highlighter:highlight:start-selected` keyed messages in the diff and was not paying attention to what the style transformation was doing, sorry about that.